File: javabasics/collections/practica3/src/itp/pk/Controller/DataGenerator.java
package itp.pk.Controller;
import itp.pk.Model.Data;
import java.text.SimpleDateFormat;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class DataGenerator {
    public static List<Data> generateData(int amount){
        List<Data> list = new ArrayList<>();;

        for (Long i=1L; i<=amount; i++){
            Long id = i;
            String name = "John";
            String key = id + name.substring(0,3);
            String status = new SimpleDateFormat("HH:mm:ss").format(new Date());

            Data worker = new Data(id, name, key, status);
            list.add(worker);
        }

        return list;
    }
}

File:  Big Data/bootcamp/python/MySQL/Data/DataGenerator.py
from Model.Data import Data
import names


class DataGenerator():

    @staticmethod
    def generateData(amount, myList):

        for i in range(amount):
            id = i
            name = names.get_first_name()
            person = Data(id, name)

            myList.append(person.id)
            myList.append(person.name)
            myList.append(person.key)
            myList.append(person.status)
        return myList
